test/jdk/java/awt/PrintJob/PrintCheckboxTest/PrintCheckboxManualTest.java

```diff
@@ -19,87 +19,54 @@
  * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
  * or visit www.oracle.com if you need additional information or have any
  * questions.
+ * @requires (os.family == "linux")
```

Comment:
@prsadhuk The OS check is added to the copyright header by mistake, please add it to jtreg section.

Reply:
Thanks for noticing it..I was trying to test on windows for some thing so I put it outside jtreg block but forgot to restore it back..